aws/resource_aws_iam_instance_profile.go

```diff
@@ -307,6 +307,9 @@ func instanceProfileReadResult(d *schema.ResourceData, result *iam.InstanceProfi
 	if err := d.Set("arn", result.Arn); err != nil {
 		return err
 	}
+	if err := d.Set("create_date", result.CreateDate); err != nil {
```

Comment:
@bootswithdefer the `*time.Time` value needs to be turned into a `string`:

```
=== RUN   TestAccAWSIAMInstanceProfile_basic
--- FAIL: TestAccAWSIAMInstanceProfile_basic (4.90s)
    testing.go:518: Step 0 error: Error applying: 1 error(s) occurred:
        
        * aws_iam_instance_profile.test: 1 error(s) occurred:
        
        * aws_iam_instance_profile.test: Error creating IAM instance profile test: create_date: '' expected type 'string', got unconvertible type '*time.Time'
```

e.g. `result.CreateDate.Format(time.RFC3339)`